justfile

```diff
@@ -61,4 +61,4 @@ check-semgrep:
   [ "$(just print-semgrep)" = "$(jq -r .semgrep < versions.json)" ] && echo '✓ semgrep versions match' || (echo '✗ semgrep version mismatch. Run `just upgrade-semgrep` to upgrade.' && exit 1)
 
 upgrade-semgrep:
-  jq '.semgrep = $v' --arg v $(just print-semgrep) <<<$(cat versions.json) > versions.json
```

Comment:
As weird as it is, I think this pattern is actually intentional right now, as it's supposed to "upgrade" the version of semgrep being used in the monorepo. I think these commands are sort of weird and I'd rather just get rid of them because it's easy to edit the version file manually.

Reply:
The issue is that everyone might have different versions on their device which might be wrong. Then they run `upgrade-semgrep` thinking it will make theirs match with the expected monorepo version but it then updates versions.json. They might think theirs is upgraded but now if that is merged, everybody also now has to run `upgrade-semgrep` because of the mismatch. which they do but now it just sets it back to whatever version they have locally. It just adds more confusion imo.